internal/cmdexport/orchestrators/tekton.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package orchestrators
+
+import (
+	"bytes"
+	"path"
+
+	"github.com/GoogleContainerTools/kpt/internal/cmdexport/types"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+// TektonPipeline wraps a Pipeline object in Tekton.
+// @see https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md
+type TektonPipeline struct {
+	APIVersion string              `yaml:"apiVersion,omitempty"`
+	Kind       string              `yaml:",omitempty"`
+	Metadata   *TektonMetadata     `yaml:",omitempty"`
+	Spec       *TektonPipelineSpec `yaml:",omitempty"`
+	task       *TektonTask
+}
+
+func (p *TektonPipeline) Init(config *types.PipelineConfig) Pipeline {
+	p.APIVersion = "tekton.dev/v1beta1"
+	p.Kind = "Pipeline"
+	p.Metadata = &TektonMetadata{Name: "run-kpt-functions"}
```

Comment:
It looks like "run-kpt-functions" are used in multiple files, can you change it to a constant?